backend/tests/api/user_test.go

```diff
@@ -39,3 +40,121 @@ func TestGetAllUsersWorks(t *testing.T) {
 	// assert that the user returned from the database is the same as the user returned from the API
 	assert.Equal(dbUser, respUser)
 }
+
+func TestUpdateUserWorks(t *testing.T) {
```

Comment:
see this new [helper](https://github.com/GenerateNU/sac/blob/main/backend/tests/api/helpers.go#L179) I made to make this testing easier / less boilerplate for you guys!

Reply:
here is an [example test](https://github.com/GenerateNU/sac/blob/main/backend/tests/api/category_test.go#L13) for inspiration